### DISCUSSION TITLE
How to plot the "F" illuminants?

### DISCUSSION THREAD

**Discussion by blue-j** [colour-science/colour#1135] [Q&A] [answered]
colour.plotting.plot_single_illuminant_sd

It seems none of the F illuminant spectra are available to being plotted?  Expected, or issue?

J

**Comment by thomasmansencal** [comment]
Hi @blue-j,

I tested a few e.g. `colour.plotting.plot_single_illuminant_sd("FL3.4")` and `colour.plotting.plot_single_illuminant_sd("FL2")` and both worked fine.

Can you please let use know which environment your are using, i.e. `colour.utilities.describe_environment` and which called you made?

Cheers,

Thomas

**Reply by blue-j** [reply, marked as answer]
I was using "F2" and the like, not "FL2." I can't find anywhere that uses the "FL" convention - is it CIE convention?

- J

**Reply by blue-j** [reply]
(I see now that the CIE in Colorimetry (4th) uses FL)

**Comment by blue-j** [comment]
just for the record:

`import colour
from colour.plotting import plot_single_illuminant_sd
import numpy as np

plot_single_illuminant_sd("FL2")

colour.plotting.render(
    diagram_opacity=1,
    title="",
    standalone=False,
    x_ticker=True,
    y_ticker=True,
    x_label="Wavelength λ [nm]",
    y_label="Relative Power",
    axes_visible=True,
    transparent_background=True,
    filename="illuminant_spectrum_legend.svg"
)`
